tests/integration/components/bs-form-test.js

```diff
@@ -321,6 +321,46 @@ module('Integration | Component | bs-form', function (hooks) {
     assert.dom(`.${formFeedbackClass()}`).hasText('There is an error');
   });
 
+  test('Form with invalid validation shows validation errors immediately when @showValidations is true', async function (assert) {
+    let model = {};
+    this.set('model', model);
+    this.set('errors', A(['There is an error']));
+
+    this.set('form', ensureSafeComponent(ValidatingForm, this));
+    this.set('formElement', ensureSafeComponent(ValidatingFormElement, this));
+
+    await render(
+      hbs`<this.form @elementComponent={{this.formElement}} @model={{this.model}} @validate={{false}} @showValidations={{true}} as |form|><form.element @errors={{this.errors}} /></this.form>`
+    );
+
+    assert
+      .dom(formFeedbackElement())
+      .hasClass(validationErrorClass(), 'validation errors are shown when @showValidations is true');
+    assert.dom(`.${formFeedbackClass()}`).hasText('There is an error');
+  });
+
+  test('Form with invalid validation does not show validation errors when @showValidations is false', async function (assert) {
```

Comment:
Similar as above, it would be great if you could extend the test to verify that also success validation is not shown.

Reply:
same as above: added test